config/config.toml.example

```diff
@@ -400,7 +400,7 @@ capacity-mb = 1000.0
 [binlog]
 # enable to write binlog.
 # NOTE: If binlog is enabled with Kafka (e.g. arbiter cluster),
-# txn-total-size-limit should be less than 1073741824(1G) because this is the maximum size that can be handled by Kafka.
+# tidb_txn_total_size_limit should be less than 1073741824(1G) because this is the maximum size that can be handled by Kafka.
```

Comment:
Please make it clear "the sysvar tidb_txn_total_size_limit should be less than.. "